sql/src/main/java/io/crate/metadata/ReferenceInfos.java

```diff
@@ -71,13 +72,17 @@ public ReferenceInfos(Map<String, SchemaInfo> builtInSchemas,
         clusterService.add(this);
     }
 
-    @Nullable
-    public TableInfo getTableInfo(TableIdent ident) {
-        SchemaInfo schemaInfo = getSchemaInfo(ident.schema());
-        if (schemaInfo != null) {
-            return schemaInfo.getTableInfo(ident.name());
+    public TableInfo getWritableTable(TableIdent tableIdent) {
+        TableInfo tableInfo = getTableInfo(tableIdent);
+        if ((tableInfo.schemaInfo().systemSchema() && !tableInfo.schemaInfo().name().equals(BlobSchemaInfo.NAME))) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                    "The table %s is read-only. Writing to it or altering it is not supported", tableInfo.ident()));
```

Comment:
Drop is also not supported if it's a system schema, so the message may be a little bit confusing. (but it's ok for me)